<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>627</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>671</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Avtorization</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Welcome!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Registration</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>442</x>
        <y>250</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Contact_List</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>291</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter  server ip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter server port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>126</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_6</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeAndSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tryConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeApplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>selected(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>NewAccountTab(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>NewAccount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>NewServerAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setsize()</slot>
  <slot>closeApplication()</slot>
  <slot>tryConnect()</slot>
  <slot>tryConnectAgain()</slot>
  <slot>sendMessage()</slot>
  <slot>changeMessageDelivered()</slot>
  <slot>NewWindow()</slot>
  <slot>Conected()</slot>
  <slot>closeAndSend()</slot>
  <slot>NewAccountTab(QString)</slot>
  <slot>NewAccount()</slot>
  <slot>NewServerAddress()</slot>
 </slots>
</ui>
